src/modules/details/components/EditTicketForm.tsx

```diff
@@ -229,6 +229,7 @@ export const EditTicketForm = ({
             onChange={handleChange}
             error={touched.status && Boolean(errors.status)}
           >
+          {userAuth.role !== 'employee' && (
```

Comment:
please create enums for user auth role and ticket status

Reply:
in codebase they have used ROLES constant can i use that if i change constant to enum i need to change everywhere in code where they have used ROLES constant

Reply:
Yes use the constant